enterprise_access/apps/api/v1/views/subsidy_access_policy.py

```diff
@@ -478,7 +478,7 @@ def get_queryset(self):
             enterprise_customer_uuid=self.enterprise_customer_uuid,
         ).order_by('-created')
 
-    def evaluate_policies(self, enterprise_customer_uuid, lms_user_id, content_key):
+    def evaluate_policies(self, enterprise_customer_uuid, lms_user_id, content_key, skip_customer_user_check=False):
```

Comment:
[inform] Called with `skip_customer_user_check=True` when `request.user.is_staff=True` and `lms_user_id` query parameter is provided.

Reply:
Oh, yep... d'oh. I said it backwards above! Thanks for catching.